site/src/pages/DeploymentSettingsPage/IdpOrgSyncPage/IdpOrgSyncPageView.tsx

```diff
@@ -0,0 +1,354 @@
+			organization_assign_default:
+				organizationSyncSettings?.organization_assign_default ?? true,
+			mapping: organizationSyncSettings?.mapping ?? {},
+		},
+		validationSchema: validationSchema,
+		onSubmit,
+		enableReinitialize: Boolean(organizationSyncSettings),
+	});
+	const [coderOrgs, setCoderOrgs] = useState<Option[]>([]);
+	const [idpOrgName, setIdpOrgName] = useState("");
+	const organizationMappingCount = form.values.mapping
+		? Object.entries(form.values.mapping).length
+		: 0;
+
+	const getOrgNames = (orgIds: readonly string[]) => {
+		return orgIds.map(
+			(orgId) =>
+				organizations.find((org) => org.id === orgId)?.display_name || orgId,
+		);
+	};
+
+	const handleDelete = async (idpOrg: string) => {
+		const newMapping = Object.fromEntries(
+			Object.entries(form.values.mapping || {}).filter(
+				([key]) => key !== idpOrg,
+			),
+		);
+		const newSyncSettings = {
+			...form.values,
+			mapping: newMapping,
+		};
+		void form.setFieldValue("mapping", newSyncSettings.mapping);
+		form.handleSubmit();
+	};
+
+	const SYNC_FIELD_ID = "sync-field";
+	const ORGANIZATION_ASSIGN_DEFAULT_ID = "organization-assign-default";
+	const IDP_ORGANIZATION_NAME_ID = "idp-organization-name";
+
+	return (
+		<div className="flex flex-col gap-2">
+			{Boolean(error) && <ErrorAlert error={error} />}
+			<form onSubmit={form.handleSubmit}>
+				<fieldset disabled={form.isSubmitting} className="border-none">
+					<div className="flex flex-row">
+						<div className="grid items-center gap-1">
+							<Label className="text-sm" htmlFor={SYNC_FIELD_ID}>
+								Organization sync field
+							</Label>
+							<div className="flex flex-row items-center gap-5">
+								<div className="flex flex-row gap-2 w-72">
+									<Input
+										id={SYNC_FIELD_ID}
+										value={form.values.field}
+										onChange={async (event) => {
+											void form.setFieldValue("field", event.target.value);
+										}}
+									/>
+									<Button
+										className="w-20"
+										disabled={form.isSubmitting || !form.dirty}
+										onClick={(event) => {
+											event.preventDefault();
+											form.handleSubmit();
+										}}
+									>
+										Save
+									</Button>
+								</div>
+								<div className="flex flex-row items-center gap-3">
+									<Switch
+										id={ORGANIZATION_ASSIGN_DEFAULT_ID}
+										checked={form.values.organization_assign_default}
+										onCheckedChange={async (checked) => {
+											void form.setFieldValue(
+												"organization_assign_default",
+												checked,
+											);
+											form.handleSubmit();
+										}}
+									/>
+									<span className="flex flex-row items-center gap-1">
+										<Label htmlFor={ORGANIZATION_ASSIGN_DEFAULT_ID}>
+											Assign Default Organization
+										</Label>
+										<AssignDefaultOrgHelpTooltip />
+									</span>
+								</div>
+							</div>
+							<p className="text-content-secondary text-2xs m-0">
```

Comment:
I know this came in via an earlier commit, but I have a lot of concerns about having the base UI get as small as 10px. I'll have to look at the design tokens, but I feel like if it's codified there, that's a mistake and should be removed

Reply:
Also, there's a typo in the tailwind Config, which is causing slightly wonky font rendering – the value for `2xs` is currently `0.626`. The `6` should be swapped to a `5` to make sure that we're going from the base size of 16px to 10px, rather than 10.16.px

Reply:
@Parkreiner The designs for 2xs are currently 10px. Why would this be a mistake? is this an accessibility issue?